src/appleseed.shaders/include/appleseed/maya/as_maya_cms_syncolor_idt.h

```diff
@@ -60,18 +51,21 @@ color transform_color_space_to_Rec709(
     {
         transformed_color = sRGB_EOTF(input_color);
     }
-    else if (color_space == "scene-linear Rec 709/sRGB")
+    else if (color_space == "scene-linear Rec 709/sRGB" ||
```

Comment:
Where are these strings coming from, exactly? They look human-readable, i.e. they don't look like identifiers; how come we can expect exact matches?

Reply:
We have the SynColor system, that provides the working/rendering space strings, they are passed via https://github.com/appleseedhq/appleseed/blob/master/src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h#L194 , the attribute for working space.
So when using SynColor we just check the options provided in maya 2017.
These are:
 * "ACES2065-1"
 * "ACEScg"
 * "scene-linear CIE XYZ"
 * "scene-linear Rec 2020"
 * "scene-linear Rec 709/sRGB"
 * "Sharp RGB"

I just noticed a typo in a couple of identifiers in a related function, so i'll be updating that.